01-shapefile-attributes.Rmd

```diff
@@ -101,13 +102,12 @@ point <- readOGR("boundaryFiles/HARV/", "HARVtower_UTM18N")
 
 ```
 
+##Querying shapefile metadata 
 Recall from the previous lesson that shapefile metadata can include a _class_, 
```

Comment:
totally -- i'm not sure the best way to address these issue but maybe we can just adjust your PR? let's stick with ` throughout for consistency :) And let me know if these line by line comments are  hard to keep track of?